services/app-web/src/pages/SubmissionSummary/SubmissionSummary.tsx

```diff
@@ -249,58 +249,64 @@ export const SubmissionSummary = (): React.ReactElement => {
                     <DocumentWarningBanner className={styles.banner} />
                 )}
 
-                {showSubmissionApproval && (
-                    <>
-                        <Grid
-                            className={styles.approveWithdrawButtonContainer}
-                            row
-                        >
-                            <ModalOpenButton
-                                id="approval-modal-toggle-button"
+                {showSubmissionApproval &&
+                    contract.reviewStatus !== 'APPROVED' && (
```

Comment:
I considered this but that would also effect the unlock button. If Kelsey wants the unlock button disabled as part of this ticket, I'll make this change. Otherwise in a future ticket the unlock btn will be removed altogether